### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by KSXGitHub**

### CODE REVIEW

File: src/dir.rs [hunk 1]
```diff
@@ -705,11 +705,11 @@ where
 
                 match _get_dir_content(_path, depth) {
                     Ok(items) => {
-                        let mut _files = items.files;
-                        let mut _dirrectories = items.directories;
+                        let mut files = items.files;
```
Comment (webdesus) [inline src/dir.rs:708]: This is changes breaking a logic... files and files it's different variables. After remove "_" we get the append method for the new variable, but we should use old. For variable "directories" we get same result.

**Review by webdesus — COMMENTED** [review]
Thank you! But you break logic in one place :upside_down_face: Can you fix it? :smiley:

File: src/dir.rs (hunk 1 above)
Comment (KSXGitHub) [reply src/dir.rs]: I have reverted the changes.

**Comment by webdesus** [comment]
@KSXGitHub thank you :relaxed:

**Pull request merged**